spec/intelligent_foods/resources/menu_spec.rb

```diff
@@ -24,4 +24,79 @@
       end
     end
   end
+
+  describe ".find" do
+    it "returns the menu" do
+      menu_id = "2023-01-01"
+      body = build_menu_response(menu_id: menu_id)
+      response = build_response(body: body)
+      stub_api_response response: response
+
+      result = IntelligentFoods::Menu.find(menu_id)
+
+      expect(result.id).to eq(menu_id)
+    end
+
+    it "returns an instance of IntelligentFoods::Menu" do
+      menu_id = "2023-01-01"
+      body = build_menu_response(menu_id: menu_id)
+      response = build_response(body: body)
+      stub_api_response response: response
+
+      result = IntelligentFoods::Menu.find(menu_id)
+
+      expect(result).to be_an_instance_of IntelligentFoods::Menu
```

Comment:
prefer not to check for types. it's a dynamic language. consider checking for an attribute value or removing these tests